plugins/chain_plugin/include/eosio/chain_plugin/blockvault_sync_strategy.hpp

```diff
@@ -66,6 +70,7 @@ struct blockvault_sync_strategy : public sync_callback {
    }
 
    void on_block(eosio::chain::signed_block_ptr block) override final {
+      ilog("Received block number ${bn}", ("bn", block->block_num()));
```

Comment:
Do not want an info level log for every block received.

Reply:
dlog?

Reply:
throttled it to every 100 blocks